go/teams/chain.go

```diff
@@ -124,6 +124,49 @@ func (t TeamSigChainState) getUserRole(user keybase1.UserVersion) keybase1.TeamR
 	return role
 }
 
+func (t TeamSigChainState) AssertAdminRoleAt(uv keybase1.UserVersion, scl keybase1.SigChainLocation) (ret keybase1.SignatureMetadataBookends, err error) {
```

Comment:
`AssertAdminRoleAt` sounds like it would return OK if asked about a `(user, seqno:5)` where that user became admin at `seqno:4`. But the code looks like it would `"not found"`.